packages/cli-e2e/__tests__/angular.specs.ts

```diff
@@ -228,15 +228,19 @@ describe('ui:create:angular', () => {
         projectName
       );
       await serverProcessManager.killAllProcesses();
-    }, 5e3);
+    }, 5 * 60e3);
```

Comment:
If it's important to bump the timeout for each, I'd just have a const somewhere at the top fo the file, used for all tests.

Also, some tests seems to have a different timeout. Is it important ? Could we just use the same value for each ?

Reply:
> If it's important to bump the timeout for each, I'd just have a const somewhere at the top fo the file, used for all tests.
That could make sense
> Also, some tests seem to have a different timeout. Is it important? Could we just use the same value for each?
Tests have different timeout depending on the 'complexity of the task at hand.

For example, everytimes we handle processes, we need to plan extra time because we ought to go through the PowerShell layer.

A more extreme example is the big beforeAll, where we instantiate the project (i.e. run the actual CLI method).
Because it often implies one or more `npm install` or other under the hood, it's quite lengthy (around 5 to 10 minutes).

We should not have 5-10' assertions time for our `it` therefore, a 'unique' timeout is not a good idea, because if doing npm start takes 5', there might be a problem.

This make me think we should not have one timeout fit for all, because it might hide some performance issues, in the test or in the framework.